types/googlemaps/reference/map.d.ts

```diff
@@ -612,6 +612,13 @@ declare namespace google.maps {
          * occurred.
          */
         latLng: LatLng;
+        /**
+         * The corresponding native DOM event.  Developres hsould not rely on target,
+         * currentTarget, relatedTarget and path properties being defined and 
+         * consistent.  Developres should not also rely on DOM structure of hte internal
+         * implementation of Maps API.
```

Comment:
A lot of typos here.

```suggestion
         * The corresponding native DOM event.  Developers should not rely on `target`,
         * `currentTarget`, `relatedTarget` and path properties being defined and
         * consistent. Developers should also not rely on DOM structure of the internal
         * implementation of Maps API.
```

But it would probably be better if you copied the description from the official docs.